recipe/meta.yaml

```diff
@@ -8,19 +8,19 @@ source:
   fn: pytest-django-{{ version }}.tar.gz
   url:  https://pypi.io/packages/source/p/pytest-django/pytest-django-{{ version }}.tar.gz
   sha256: 038ccc5a9daa1b1b0eb739ab7dce54e495811eca5ea3af4815a2a3ac45152309
+  patches:
+    # setup_requires is throwing errors in setuptools
+    - setuptools_scm.patch
 
 build:
-  number: 0
-  preserve_egg_dir: True
```

Comment:
What was this needed for?

Reply:
What was the removal of `preserve_egg_dir: True` needed for? Simply because it wasn't required any longer.

Reply:
Was meaning what was `preserve_egg_dir` doing for us previously that helped this package?

Reply:
ah-ha! My best guess is that it wouldn't build for me without it in the original `staged-recipes` PR. Most likely had something to do with the use of `setuptools_scm` which is always a pain to deal with.